README.md

```diff
@@ -0,0 +1,42 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio: mobile version
+
+> This project is Portfolio mobile version.
+
```

Comment:
[OPTIONAL]

- [ ] Please, always commit your work in the present tense. All your commit messages have been done in the past tense. Kindly take note of this. :pray:

Reply:
Thank you.